src/main/java/se/sundsvall/supportmanagement/integration/db/model/CommunicationAttachmentEntity.java

```diff
@@ -54,11 +56,13 @@ public class CommunicationAttachmentEntity {
 	@Column(name = "file_size")
 	private Integer fileSize;
 
-	@OneToOne(fetch = FetchType.LAZY, cascade = CascadeType.ALL, orphanRemoval = true)
-	@JoinColumn(name = "communication_attachment_data_id", nullable = false, foreignKey = @ForeignKey(name = "fk_communication_attachment_data_communication_attachment"))
-	private CommunicationAttachmentDataEntity attachmentData;
+	@ManyToOne(fetch = LAZY, cascade = {
+		PERSIST, MERGE, REFRESH
+	})
+	@JoinColumn(name = "attachment_data_id", nullable = false, foreignKey = @ForeignKey(name = "fk_communication_attachment_data_attachment"))
```

Comment:
Should this be named: `fk_communication_attachment_attachment_data`

(I suppose the pattern is: `fk_<table>_<referenced_table>` ?